### ISSUE TITLE
File log viewer not working anymore

### ISSUE THREAD

**Issue by kipusoep** [DanDiplo/UmbracoTraceLogViewer#12]
Since the filename of the log files has been changed to include the server's name, the Log viewer dashboard isn't able to show the contents of the 'File' tab anymore.

**Comment by DanDiplo** [comment]
@kipusoep I'm afraid I'm not clear what you mean. What is the "File" tab you refer to? I don't recall any tab!

The logviewer has continued functioning since the log file names were renamed - I'm running it currently on 7.3.8 and it works just the same as before using latest NuGet package.

So I'm thinking we are at cross purposes. Can you include a screenshot of what you mean, please? Also what version of Umbraco you are testing against. Thanks!

**Comment by kipusoep** [comment]
Okay this is awkward, it's an issue with some othe Log viewer plug-in :-P Sorry!

**Comment by DanDiplo** [comment]
@kipusoep Well, if you will flirt with other girls you get what you deserve :stuck_out_tongue:

**Comment by kipusoep** [comment]
It was this one: https://github.com/lars-erik/Umbraco.LogViewer
AFAIK the main difference is that it's a live log viewer, does yours support 'live' too? I'd like to switch because the other one looks dead...

**Comment by DanDiplo** [comment]
Erm, it will if you hit "reload" :)

I did once actually have an experimental auto-update feature where it would auto-refresh whenever a new entry was detected. It used HTML5 websockets to "push" changes, but I found them not to be too reliable and difficult to get working in .NET on the server side.

I think adding a more simple client-side polling version might be simpler. So basically the client-side Angular script would check every 10 secs to see if the log file has changed (based on timestamp or filesize) and then update based on that. So I could look at that when I have time...
